src/routes.ts

```diff
@@ -1,9 +1,15 @@
-import express from 'express';
-import item from './controller/item'
+import express from 'express'
+import task from './controller/task'
 
-const router = express.Router();
+const router = express.Router()
 
-router.route('/items')
-  .post(item.createItem)
+router.route('/tasks')
+  .post(task.createTask)
+  .get(task.getAllTasks)
 
-export default router
\ No newline at end of file
+router.route('/tasks/:id')
```

Comment:
Please, add PATCH method as well.

Reply:
method PATCH for updating task staus /task importance added

Reply:
30afaebf219a08e95436d413662504fff035dd68